### PR TITLE
ODC-7400: Fix e2e for dev-console, knative, topology, helm and pipeline packages

### PR DESCRIPTION
**Pull request by vikram-raj**
Story: [ODC-7400](https://issues.redhat.com//browse/ODC-7400)

Descriptions: Enable and fix e2e tests post cypress upgrade

### CODE REVIEW

**Comment by sanketpathak** [comment]
@vikram-raj Please add this `topologyPage.verifyWorkloadInTopologyPage(helmReleaseName);` before this [line](https://github.com/openshift/console/blob/aeeeddf947e2c5733b162b3fb17c8c06a6bdc8e5/frontend/packages/helm-plugin/integration-tests/support/step-definitions/helm/helm-release.ts#L29) the failed tests will work

File: frontend/packages/dev-console/integration-tests/plugins/index.js [hunk 1]
```diff
@@ -86,13 +86,13 @@ module.exports = (on, config) => {
...
   // eslint-disable-next-line global-require
-  const configJson = require(config.configFile);
-  if (configJson.extends) {
-    const baseConfigFilename = path.join(config.projectRoot, configJson.extends);
-    // eslint-disable-next-line import/no-dynamic-require
-    const baseConfig = require(baseConfigFilename);
-    console.log('merging %s with %s', baseConfigFilename, config.configFile);
-    return deepmerge(baseConfig, configJson);
-  }
+  // const configJson = require(config.configFile);
```
Comment (rhamilto) [inline frontend/packages/dev-console/integration-tests/plugins/index.js:89]: Here and other commented code elsewhere:  go ahead and remove if not necessary.

**Review by rhamilto — APPROVED** [review]
Nit regarding removing commented code.  Otherwise LGTM

**Review by rhamilto — APPROVED** [review]
/lgtm

**Comment by vikram-raj** [comment]
No doc and UX changes

/label docs-approved
/label px-approved

**Pull request merged**